client/src/Movies/SavedList.js

```diff
@@ -10,9 +12,13 @@ export default class SavedList extends Component {
       <div className="saved-list">
         <h3>Saved Movies:</h3>
         {this.props.list.map(movie => (
+          <NavLink exact to={`/movie/${movie.id}`}>
```

Comment:
I want you to never forget our rule of thumb in React after using the map method
We will always set a key to the first element after map
```suggestion
          <NavLink exact to={`/movie/${movie.id}`} key={movie.id + Date.now()}>
```